src/xcode/ENA/ENA/Source/Extensions/NSAttributedString+BulletPoint.swift

```diff
@@ -15,7 +15,7 @@ extension NSAttributedString {
 	///   - from: The initial attributed string.
 	///   - bulletPointFont: The Font for the bullet point and first part of the `from` string. Required to align and scale the bullet point.
```

Comment:
Can we have a description here, what `overwriteAttributes ` is doing?

Reply:
Done. Forget that place